### ISSUE TITLE
"Extend" on spanish translation

### ISSUE THREAD

**Issue by distriker** [anchorcms/anchor-translations#71]
Hi everyone,
I trying to complete the spanish translation. But I have a doubt about "**extend**".

In spanish, "extend" is "extendido" o "ampliado". But, in "extend" is where Anchor has its configuration. Perhaps, a better translation is "**configuración**" (configuration).

Don't you think?

Regards, _Iván_

**Comment by TimohStudios** [comment]
I agree with you in this doubt. However, I can't really see why it says "Extend" in the first place. I'm not a native English speaker myself, but I would think that configuration would make more sense in English as well. Just sticking to this issue, sticking with whatever translates closest to Extend in this case is my recommendation, since that is what I found in the other translations that I could understand (although the latest Dutch PR contained a typo and the French one seemed to go with something like Extensions). In the future it would be a more general issue to discuss if Extend really is a name to stick with.

That's just my opinion though.
